### PR TITLE
zh,en: add task-configuration-file-full to TOC

### PR DESCRIPTION
**Pull request by uglyengineer**
### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by uglyengineer** [comment]
@yikeke

File: en/TOC.md [hunk 1]
```diff
@@ -68,6 +68,7 @@
     - [DM-worker Configuration](dm-worker-configuration-file.md)
     - [Upstream Database Configuration](source-configuration-file.md)
     - [Task Configuration](task-configuration-file.md)
+    - [Full Task Configuration](task-configuration-file-full.md)
```
Comment (yikeke) [inline en/TOC.md:71]: The two documents have a similar structure. Can we merge the two files into one? @WangXiangUSTC @csuzhangxc @toutdesuite

Comment (csuzhangxc) [reply en/TOC.md:71]: in most cases, the user only needs to read _task-configuration-file.md_, so we split into two docs to make it clearer.

Comment (yikeke) [reply en/TOC.md:71]: Ok. I guess we can leave it be for now.

**Pull request merged**